src/NHapi.Base/Parser/EncodingCharacters.cs

```diff
@@ -52,13 +52,14 @@ public class EncodingCharacters : object, ICloneable
         /// Component Separator, Repetition Separator, Escape Character, and
         /// Subcomponent Separator (in that order).
         /// </param>
+        /// <exception cref="HL7Exception">If encoding characters are not unique.</exception>
         public EncodingCharacters(char fieldSeparator, string encodingCharacters)
         {
             FieldSeparator = fieldSeparator;
 
             encChars = new char[4];
 
-            if (encodingCharacters == null)
+            if (string.IsNullOrEmpty(encodingCharacters))
```

Comment:
do we need to check whitespace?

Reply:
im not 100% sure but I would say no because they might want to use a whitespace character such as `\n` (this is an assumption). - what do you think?

Reply:
yeah, could be valid - can leave for now and revisit if we need to?

Reply:
![image](https://user-images.githubusercontent.com/1087376/120340859-47f25180-c2ee-11eb-9c2c-9d31b41e1d7c.png)
Will perform whitespace checks and validations.